ocaml/xapi/storage_migrate.ml

```diff
@@ -300,7 +300,9 @@ let copy' ~task ~dbg ~sr ~vdi ~url ~dest ~dest_vdi =
 				);
 			)
 			(fun () -> 
-				Remote.DP.destroy ~dbg ~dp:remote_dp ~allow_leak:false);
+				Remote.DP.destroy ~dbg ~dp:remote_dp ~allow_leak:false;
+				State.remove id State.active_copy
```

Comment:
Its getting added at https://github.com/xenserver/xen-api/blob/cream-bugfix/ocaml/xapi/storage_migrate.ml#L278
This is same as in trunk https://github.com/xapi-project/xen-api/blob/master/ocaml/xapi/storage_migrate.ml#L362